django/contrib/admin/static/admin/css/base.css

```diff
@@ -187,9 +187,15 @@ p.mini {
     margin-top: -3px;
 }
 
-.help, p.help, form p.help {
+.help, span.help, form span.help, form span.help li {
     font-size: 11px;
     color: #999;
+    display: block;
+}
+
+form span.help ul {
+    margin-left: 0px !important;
```

Comment:
There's a ticket about removing `!important` from the admin's CSS, so we should try not to introduce more of that.
Also "0px" should be "0".